cmd/podman/manifest/inspect.go

```diff
@@ -25,6 +25,10 @@ func init() {
 		Command: inspectCmd,
 		Parent:  manifestCmd,
 	})
+	flags := inspectCmd.Flags()
+
+	flags.Bool("insecure", false, "Purely for Docker compatibility")
```

Comment:
This seems like it should force `--tls-verify=force` instead of doing nothing

Reply:
tls-verify is not supported yet for podma manifest inspect, that I can see.

Reply:
I am not sure it even talks to a remote service.

@nalind WDYT?

Reply:
It connects to a registry if there's no local list with the specified name, I think since #7735.

Reply:
Yes, that sounds good.  `--insecure` can then also just alias to `--tls-verify`.

Reply:
Requires passing the value down to `remoteManifestInspect()` and flip the switch in the system context.